OpenEdXMobile/src/main/java/org/edx/mobile/view/DiscoveryActivity.java

```diff
@@ -41,18 +42,23 @@ public Fragment getFirstFragment() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setTitle(R.string.label_discover);
         environment.getAnalyticsRegistry().trackScreenView(Analytics.Screens.FIND_COURSES);
     }
 
     @Override
-    public void onResume() {
-        super.onResume();
-        AuthPanelUtils.configureAuthPanel(findViewById(R.id.auth_panel), environment);
+    protected void configureActionBar() {
+        ActionBar bar = getSupportActionBar();
+        if (bar != null) {
+            bar.hide();
+            bar.setDisplayShowHomeEnabled(false);
+            bar.setDisplayHomeAsUpEnabled(false);
+            bar.setIcon(android.R.color.transparent);
+        }
     }
 
     @Override
-    public void setTitle(int titleId) {
-        setTitle(getResources().getString(titleId));
+    public void onResume() {
```

Comment:
why the change is effecting this method?

Reply:
Locally it's not showing this as a change. But let me look into it.